src/redux/Destinations/destinations.js

```diff
@@ -1,19 +1,28 @@
-import { getDestinations } from '../../helpers/getDestinationData';
+const GET_DESTINATIONS = 'GET_DESTINATIONS';
 
-const FETCH_DATA = 'FETCH_DATA';
-const initialState = [];
+const initialValue = {};
 
-export const fetchDestinations = () => async (dispatch) => {
-  const destinations = await getDestinations();
-  dispatch({ type: FETCH_DATA, payload: destinations });
+const getDestination = (payload) => ({
+  type: GET_DESTINATIONS,
+  payload,
+});
+
+const getDestinationThunk = () => async (dispatch) => {
+  const request = await fetch('http://localhost:3000/api/v1/destinations');
+  const data = await request.json();
+  dispatch(getDestination(data));
 };
```

Comment:
[Optional]
Please consider moving this fetching destinations functionality into helper functions. We already created this architecture, so it will be nice that we all follow that pattern.